src/components/file-upload.tsx

```diff
@@ -362,74 +340,87 @@ export default class FileUpload extends React.Component {
   }
 
   //Stores users input for manually entered table to allow for persistence across page changes
-  handleManualEarnings(input) {
-    // eslint-disable-next-line
-    const [type, year, key] = input.target.id.split("_");
-
+  handleManualEarnings = (year: string, value: string) => {
     var tempManualTable = this.state.manualTable;
 
     if (Object.keys(tempManualTable).includes(year)) {
-      tempManualTable[year] = Number(input.target.value);
+      tempManualTable[year] = Number(value);
     }
 
     this.setState({
       manualTable: tempManualTable
     });
   }
 
+  showSaveMessage = () => {
+    const saveMessageEl = this.saveMessageRef.current
+    if (saveMessageEl) {
+      saveMessageEl.style.display = "grid";
+      setTimeout(function() {
+        saveMessageEl.style.display = "none";
+      }, 3000);
+    }
+  }
+
   //Saves manually entered record to this.state.earningsRecord object, becomes noticable to Observable API
-  handleSave() {
+  handleSave = () => {
+    const {
+      userState: {earnings},
+      userStateActions: {setEarnings},
+    } = this.props
     //Load earnings record; if not present, set default record.
-    var tempRecord = this.state.earningsRecord ? this.state.earningsRecord : {};
+    var tempRecord = earnings ?? {}
```

Comment:
@paavanb oops, looks like it needs a little tweak to build yet.
```
10:31:35 PM: /opt/build/repo/src/components/file-upload.tsx: Support for the experimental syntax 'nullishCoalescingOperator' isn't currently enabled (372:31):
10:31:35 PM:   370 |     } = this.props
10:31:35 PM:   371 |     //Load earnings record; if not present, set default record.
10:31:35 PM: > 372 |     var tempRecord = earnings ?? {}
```

Reply:
How do we enable that tweak?